### PR TITLE
Bug fix, minor cleanup, and additional scripts migrated from STDcheck

### PR DESCRIPTION
**Pull request by Chekote**
Please check the individual commits for details.

### CODE REVIEW

File: lib/akita.bash [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+    docker container ls --format "{{.Image}} {{.Names}}" | grep "$AKITA_IMAGE" | awk '{print $2}' | xargs docker kill --signal=SIGINT
+
+    ###
+    # Temporarily disabled this, as it causes an error in CI and we are working with Akita to fix it
+    #
+    # .circleci/bin/akita/stop_akita.sh: line 11: wait: pid 16066 is not a child of this shell
+    # ##
+    # wait "$SUPER_LEARN_PID"
+}
```
Comment (Vagoasdf) [inline lib/akita.bash:90]: Shouldn't this have an extra line at the end?

File: lib/circleci.sh [hunk 2]
```diff
@@ -43,6 +48,141 @@ circleci.build.cancel() {
...
+    : "${CIRCLE_BUILD_NUM:?"Required Env Variable not found!"}"
+    : "${CIRCLE_PROJECT_USERNAME:?"Required Env Variable not found!"}"
+    : "${CIRCLE_PROJECT_REPONAME:?"Required Env Variable not found!"}"
+    : "${CIRCLE_REPOSITORY_URL:?"Required Env Variable not found!"}"
+
+    if [ -z "${1:-}" ]; then
+        echo "Must provide Max Queue Time in *minutes* as script argument"
+        exit 1
+    fi
+    max_time="$1"
```
Comment (BKorenek) [inline lib/circleci.sh:132]: We should assign this locally correct?

File: lib/circleci.sh [hunk 3]
```diff
@@ -43,6 +48,141 @@ circleci.build.cancel() {
...
+# Environment Variables:
+#   CIRCLE_TOKEN            The auth token for the CircleCI API
+#   CIRCLE_BUILD_NUM        The number of the current job. Job numbers are unique for each job.
+#   CIRCLE_PROJECT_USERNAME The GitHub or Bitbucket username of the current project.
+#   CIRCLE_PROJECT_REPONAME The name of the repository of the current project.
+#   CIRCLE_REPOSITORY_URL   The URL of your GitHub or Bitbucket repository.
+#
+# Arguments:
+#   1 The maximum queue time in minutes.
+#   1 The name of the job that is queueing.
```
Comment (BKorenek) [inline lib/circleci.sh:118]: two 1's

File: lib/akita.bash [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+# Environment Variables:
+#   AKITA_IMAGE The Docker tag for the image that the container is using
+#######################################
+akita.stop () {
+    if [ ! -v AKITA_IMAGE ]; then
+      echo "The AKITA_IMAGE env var must be set"
+      return 1
+    fi
+
+    docker container ls --format "{{.Image}} {{.Names}}" | grep "$AKITA_IMAGE" | awk '{print $2}' | xargs docker kill --signal=SIGINT
```
Comment (BKorenek) [inline lib/akita.bash:82]: for documentation sake shouldn't we extract this $2? I'm not actually sure if this is an argument to the function or some other bash magic really. 

If it is an actual argument we would also need documentation

File: lib/mergrefreeze.bash [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+#   1 The username of the account that hosts the github project
+#   2 The name of the repository to be frozen
+#######################################
+mergefreeze.freeze(){
+  local username="$1"
+  local reponame="$2"
+
+  local url
+  url="$(mergefreeze.url "$username" "$reponame")"
+  local username="$1"
```
Comment (BKorenek) [inline lib/mergrefreeze.bash:38]: Double declaration i think

**Review by BKorenek — CHANGES_REQUESTED** [review]

File: lib/akita.bash (hunk 1 above)
Comment (Chekote) [reply lib/akita.bash:90]: Yes, it should!

**Review by BKorenek — APPROVED** [review]

**Pull request merged**